docs/package-types/esm-view.md

```diff
@@ -0,0 +1,154 @@
+---
+parent: Package Types
+nav_order: 20
+title: esm-view
+---
+
+# ESM View
+
+Modular `esm-view`s are built with the same Webpack or esbuild configuration and
+support the same functionalities as the [`app`](./app.md) type, with the
```

Comment:
Makes me wonder why it's not called "esm-app", but let's not go there!

Reply:
Technically it is an app (it can be standalone) **and** a view (it can be a separate auxiliary module loaded on-demand by a standalone SPA). In retrospective, I would have called it "micro-frontend".

Reply:
I think the most compelling use-case of Modular going forward will be its support of a micro-frontend build and deployment pattern. In that vein, my current thinking is that the distinction between "apps" and "views" goes away and we can go back to calling everything an "app" because I predict that the most common use-cases will look like standalone apps or apps with several different information screens that are themselves relatively self-contained. The term "app" gets a little meaningless when the apps get really small, but I don't expect that will happen very often.